crates/dwallet-classgroups-types/src/lib.rs

```diff
@@ -51,27 +49,74 @@ impl ClassGroupsKeyPairAndProof {
     }
 }
 
-/// Generates a class groups key pair, and proof that
-/// the generated public key is a class groups key.
-/// The keypair is generated from a given seed
-/// by initiating a seed-based random number generator with it.
+/// Generate a class groups keypair and proof from a seed.
```

Comment:
fix the doc explain when the feature is used and when it is unused and what is the diff